File: bcci.vb
Imports System

Module Program

    Sub Main()
        Dim ContinueInquiry As Char

        Do
            Console.Clear()
            Dim EmployeeName As String
            Dim CompanyName As String
            Dim MonthlySalary As Double
            Dim TaxResult As Double
            Dim SSSResult As Double
            Dim PagIbigResult As Double
            Dim PhilHealthResult As Double

            Console.WriteLine("=========================== WELCOME TO BCCI! ===========================" & vbCrLf)
            Console.Write("NAME OF EMPLOYEE: ")
            EmployeeName = Console.ReadLine()
            Console.Write("COMPANY NAME: ")
            CompanyName = Console.ReadLine()
            Console.Write("MONTHLY SALARY: ₱ ")
            MonthlySalary = Convert.ToDouble(Console.ReadLine())

            TaxResult = IncomeTax(MonthlySalary)
            Console.WriteLine(vbCrLf & vbTab & vbTab & "INCOME TAX: ₱ " & (TaxResult / 12).ToString("F2"))

            SSSResult = SSSContribution(MonthlySalary)
            Console.WriteLine(vbCrLf & vbTab & vbTab & "SSS CONTRIBUTION: ₱ " & SSSResult.ToString("F2"))

            PagIbigResult = PagIbigContribution(MonthlySalary)
            Console.WriteLine(vbCrLf & vbTab & vbTab & "PAG-IBIG: ₱ " & PagIbigResult.ToString("F2"))

            PhilHealthResult = PhilHealthContribution(MonthlySalary)
            Console.WriteLine(vbCrLf & vbTab & vbTab & "PHILHEALTH: ₱ " & PhilHealthResult.ToString("F2"))

            Console.Write(vbCrLf & vbCrLf & "DO YOU WANT TO MAKE ANOTHER INQUIRY [Y/N]? ")
            ContinueInquiry = Console.ReadLine().ToUpper()(0)
        Loop While ContinueInquiry = "Y"

    End Sub

    ' INCOME TAX
    Function IncomeTax(MonthlySalary As Double) As Double
        Dim annualSalary As Double = MonthlySalary * 12
        Dim resultTax As Double = 0

        If annualSalary < 250000 Then
            resultTax = 0
        ElseIf annualSalary >= 250000 AndAlso annualSalary < 400000 Then
            resultTax = (annualSalary - 250000) * 0.2
        ElseIf annualSalary >= 400000 AndAlso annualSalary < 800000 Then
            resultTax = (annualSalary - 400000) * 0.25 + 30000
        ElseIf annualSalary >= 800000 AndAlso annualSalary < 2000000 Then
            resultTax = (annualSalary - 800000) * 0.3 + 130000
        ElseIf annualSalary >= 2000000 AndAlso annualSalary < 8000000 Then
            resultTax = (annualSalary - 2000000) * 0.32 + 490000
        Else
            resultTax = (annualSalary - 8000000) * 0.35 + 2410000
        End If

        Return resultTax
    End Function

    ' SSS CONTRIBUTION
    Function SSSContribution(MonthlySalary As Double) As Double
        Dim salary As Double = MonthlySalary
        Dim contribution As Double

        Select Case salary
            Case < 2250
                contribution = 80
            Case 2250 To 2750
                contribution = 100
            Case 2750 To 3250
                contribution = 120
            Case 3250 To 3750
                contribution = 100
            Case 3750 To 4250
                contribution = 100
            Case 4250 To 4750
                contribution = 100
            Case 4750 To 5250
                contribution = 100
            Case 5250 To 5750
                contribution = 100
            Case 5750 To 6250
                contribution = 100
            Case 6250 To 6750
                contribution = 100
            Case 6750 To 7250
                contribution = 100
            Case 7250 To 7750
                contribution = 100
            Case 7750 To 8250
                contribution = 100
            Case 8250 To 8750
                contribution = 100
            Case 8750 To 9250
                contribution = 100
            Case 9250 To 9750
                contribution = 100
            Case 9750 To 10250
                contribution = 100
            Case 10250 To 10750
                contribution = 100
            Case 10750 To 11250
                contribution = 100
            Case 11250 To 11750
                contribution = 100
            Case 11750 To 12250
                contribution = 100
            Case 12250 To 12750
                contribution = 100
            Case 12750 To 13250
                contribution = 100
            Case 13250 To 13750
                contribution = 100
            Case 13750 To 14250
                contribution = 100
            Case 14250 To 14750
                contribution = 100
            Case 14750 To 15250
                contribution = 100
            Case 15250 To 15750
                contribution = 100
            Case 15750 To 16250
                contribution = 100
            Case 16250 To 16750
                contribution = 100
            Case 16750 To 17250
                contribution = 100
            Case 17250 To 17750
                contribution = 100
            Case 17750 To 18250
                contribution = 100
            Case 18250 To 18750
                contribution = 100
            Case 18750 To 19250
                contribution = 760
            Case 19250 To 19750
                contribution = 780
            Case Else
                contribution = 800
        End Select

        Return contribution
    End Function

    ' PAG-IBIG
    Function PagIbigContribution(MonthlySalary As Double) As Double
        Dim contribution As Double

        If MonthlySalary <= 1500 Then
            contribution = MonthlySalary * 0.01
        Else
            contribution = MonthlySalary * 0.02
        End If

        Return contribution
    End Function

    ' PHILHEALTH
    Function PhilHealthContribution(MonthlySalary As Double) As Double
        Dim salary As Double = MonthlySalary
        Dim contribution As Double

        Select Case salary
            Case < 8999.99
                contribution = 100
            Case 9000.0 To 9999.99
                contribution = 112.5
            Case 10000.0 To 10999.99
                contribution = 125
            Case 11000.0 To 11999.99
                contribution = 137.5
            Case 12000.0 To 12999.99
                contribution = 150
            Case 13000.0 To 13999.99
                contribution = 162.5
            Case 14000.0 To 14999.99
                contribution = 175
            Case 15000.0 To 15999.99
                contribution = 187.5
            Case 16000.0 To 16999.99
                contribution = 200
            Case 17000.0 To 17999.99
                contribution = 212.5
            Case 18000.0 To 18999.99
                contribution = 225
            Case 19000.0 To 19999.99
                contribution = 237.5
            Case 20000.0 To 20999.99
                contribution = 250
            Case 21000.0 To 21999.99
                contribution = 262.5
            Case 22000.0 To 22999.99
                contribution = 275
            Case 23000.0 To 23999.99
                contribution = 287.5
            Case 24000.0 To 24999.99
                contribution = 300
            Case 25000.0 To 25999.99
                contribution = 312.5
            Case 26000.0 To 26999.99
                contribution = 325
            Case 27000.0 To 27999.99
                contribution = 337.5
            Case 28000.0 To 28999.99
                contribution = 350
            Case 29000.0 To 29999.99
                contribution = 362.5
            Case 30000.0 To 30999.99
                contribution = 375
            Case 31000.0 To 31999.99
                contribution = 387.5
            Case 32000.0 To 32999.99
                contribution = 400
            Case 33000.0 To 33999.99
                contribution = 412.5
            Case 34000.0 To 34999.99
                contribution = 425
            Case Else
                contribution = 437.5
        End Select

        Return contribution
    End Function

End Module

File: bcci.c
#include<stdio.h>
#include<stdlib.h>
float PersonalIncomeTax(float as);
float SSSContri(float ms);
float PagIbig(float ms);
float PhilHealth(float ms);

int main()
{
    int ms,as;
    float sss,ph,tax,pi;
    char name[30], nameCompany[30],choose;
do 
{
    printf("NAME OF EMPLOYEE: ");
    scanf("%s", name);
    fgets(name,30,stdin);
    printf("NAME OF COMPANY: ");
    fgets(nameCompany,30,stdin);
    printf("MONTHLY SALARY: ");
    scanf("%d", &ms);
    
    as=ms*12;
    tax=PersonalIncomeTax(as);
    sss=SSSContri(ms);
    pi=PagIbig(ms);
    ph=PhilHealth(ms);
    
    printf("\n\tINCOME TAX: %.2f", tax);
    printf("\n\tSSS CONTRIBUTION: %.2f", sss);
    printf("\n\tPAG-IBIG: %.2f", pi);
    printf("\n\tPHILHEALTH: %.2f", ph);
    
    printf("\n\nANOTHER INQUIRY[Y/N]? ");
    scanf("%s", &choose);
    system("clear");
    
}while(choose=='Y'|| choose=='y');
system("clear");
printf("THANK YOU FOR USING THE BCCI SYSTEM BY JUGO");
printf("\n\n\tCOME AGAIN!! ");
}

float PersonalIncomeTax(float as)
{
    float tax;
    if(as<250000) 
    {
    tax=0.00;
    return tax;
    } else if (as>250000&&as<=400000) {
    tax=(.20*(as-250000))/12;
    return tax;
    } else if (as>400000&&as<=800000) {
    tax=(30000+.25*(as-400000))/12;
    return tax;
    } else if (as>800000&&as<=2000000) {
    tax=(130000+.30*(as-800000))/12;
    return tax;
    } else if (as>2000000&&as<=8000000) {
    tax=(490000+.32*(as-2000000))/12;
    return tax;
    } else {
    tax=(2410000+.35*(as-8000000))/12;
    return tax;
    }
    }
float SSSContri(float ms)
{
    float sss;
    if (ms<2250) {
    sss=80.00;
    return sss;
    } else if (ms>2250&&ms<=2749) {
    sss=100.00;
    return sss;
    } else if (ms>2750&&ms<=3249) {
    sss=120.00;
    return sss;
    } else if (ms>3250&&ms<=3749) {
    sss=140.00;
    return sss;
    } else if (ms>3750&&ms<=4249) {
    sss=160.00;
    return sss;
    } else if (ms>4250&&ms<=4749) {
    sss=180.00;
    return sss;
    } else if (ms>4750&&ms<=5249) {
    sss=200.00;
    return sss;
    } else if (ms>5250&&ms<=5749) {
    sss=220.00;
    return sss;
    } else if (ms>5750&&ms<=6249) {
    sss=240.00;
    return sss;
    } else if (ms>6250&&ms<=6749) {
    sss=260.00;
    return sss;
    } else if (ms>6750&&ms<=7249) {
    sss=280.00;
    return sss;
    } else if (ms>7250&&ms<=7749) {
    sss=300.00;
    return sss;
    } else if (ms>7750&&ms<=8249) {
    sss=320.00;
    return sss;
    } else if (ms>8250&&ms<=8749) {
    sss=340.00;
    return sss;
    } else if (ms>8750&&ms<=9249) {
    sss=360.00;
    return sss;
    } else if (ms>9250&&ms<=9749) {
    sss=380.00;
    return sss;
    } else if (ms>9750&&ms<=10249) {
    sss=400.00;
    return sss;
    } else if (ms>10250&&ms<=10749) { 
    sss=420.00;
    return sss;
    } else if (ms>10750&&ms<=11249) {
    sss=440.00;
    return sss;
    } else if (ms>11250&&ms<=11749) {
    sss=460.00;
    return sss;
    } else if (ms>11750&&ms<=12249) {
    sss=460.00;
    return sss;
    } else if (ms>12250&&ms<=12749) {
    sss=480.00;
    return sss;
    } else if (ms>12750&&ms<=13249) {
    sss=500.00;
    return sss;
    } else if (ms>13250&&ms<=13749) {
    sss=520.00;
    return sss;
    } else if (ms>13750&&ms<=14249) {
    sss=540.00;
    return sss;
    } else if (ms>14250&&ms<=14749) {
    sss=560.00;
    return sss;
    } else if (ms>14750&&ms<=15249) {
    sss=580.00;
    return sss;
    } else if (ms>15250&&ms<=15749) {
    sss=600.00;
    return sss;
    } else if (ms>15750&&ms<=16249) {
    sss=620.00;
    return sss;
    } else if (ms>16250&&ms<=16749) {
    sss=640.00;
    return sss;
    } else if (ms>16750&&ms<=17249) {
    sss=680.00;
    return sss;
    } else if (ms>17250&&ms<=17749) {
    sss=700.00;
    return sss;
    } else if (ms>17750&&ms<=18249) {
    sss=720.00;
    return sss;
    } else if (ms>18250&&ms<=18749) {
    sss=740.00;
    return sss;
    } else if (ms>18750&&ms<=19249) {
    sss=760.00;
    return sss;
    } else if (ms>19250&&ms<=19749) {
    sss=780.00;
    return sss;
    } else {
    sss=800.00;
    return sss;
    }
}
float PagIbig(float ms)
{
    float pi;
    if(ms<=1500) {
    pi=.01*ms;
    return pi;
    } else {
    pi=.02*ms;
    return pi;
    }
    }
float PhilHealth(float ms)
{
    float ph;
    if (ms<8999) {
    ph=100.00;
    return ph;
    } else if (ms>9000&&ms<=9999) {
    ph=112.50;
    return ph;
    } else if (ms>10000&&ms<=10999) {
    ph=125.00;
    return ph;
    } else if (ms>11000&&ms<=11999) {
    ph=137.50;
    return ph;
    } else if (ms>12000&&ms<=12999) {
    ph=150.00;
    return ph;
    } else if (ms>13000&&ms<=13999) {
    ph=162.50;
    return ph;
    } else if (ms>14000&&ms<=14999) {
    ph=175.00;
    return ph;
    } else if (ms>15000&&ms<=15999) {
    ph=187.50;
    return ph;
    } else if (ms>16000&&ms<=16999) {
    ph=200.00;
    return ph;
    } else if (ms>17000&&ms<=17999) {
    ph=212.50;
    return ph;
    } else if (ms>18000&&ms<=18999) {
    ph=225.00;
    return ph;
    } else if (ms>19000&&ms<=19999) {
    ph=237.50;
    return ph;
    } else if (ms>20000&&ms<=20999) {
    ph=250.00;
    return ph;
    } else if (ms>21000&&ms<=21999) {
    ph=262.50;
    return ph;
    } else if (ms>22000&&ms<=22999) {
    ph=275.00;
    return ph;
    } else if (ms>23000&&ms<=23999) {
    ph=287.50;
    return ph;
    } else if (ms>24000&&ms<=24999) {
    ph=300.00;
    return ph;
    } else if (ms>25000&&ms<=25999) {
    ph=312.50;
    return ph;
    } else if (ms>26000&&ms<=26999) {
    ph=325.00;
    return ph;
    } else if (ms>27000&&ms<=27999) {
    ph=337.50;
    return ph;
    } else if (ms>28000&&ms<=28999) {
    ph=350.00;
    return ph;
    } else if (ms>29000&&ms<=29999) {
    ph=362.50;
    return ph;
    } else if (ms>30000&&ms<=30999) {
    ph=375.00;
    return ph;
    } else if (ms>31000&&ms<=31999) {
    ph=387.50;
    return ph;
    } else if (ms>32000&&ms<=32999) {
    ph=400.00;
    return ph;
    } else if (ms>33000&&ms<=33999) {
    ph=412.50;
    return ph;
    } else if (ms>34000&&ms<=34999) {
    ph=425.00;
    return ph;
    } else {
    ph=437.50;
    return ph;
    }
    }

